tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -174,3 +195,17 @@ The possible statuses of a private endpoint service are explained as follows:
 - **Creating**: The endpoint service is being created, which takes 3 to 5 minutes.
 - **Active**: The endpoint service is created, no matter whether the private endpoint is created or not.
 - **Deleting**: The endpoint service or the cluster is being deleted, which takes 3 to 5 minutes.
+
+## Troubleshooting
+
+### I cannot connect to TiDB cluster via private endpoint after enabling private DNS. Why?
+
+You might need to properly set the security group for your VPC endpoint in the AWS Management Console. Go to **VPC** > **Endpoints**. Right-click your VPC endpoint and select **Manage security groups**.
+
+In the **Manage security groups** page, properly select the security group for your VPC endpoint.
```

Comment:
This sentence is a bit redundant and can be deleted.

Reply:
agree